core/templates/dev/head/components/OutcomeEditorDirective.js

```diff
@@ -63,10 +67,6 @@ oppia.directive('outcomeEditor', [
             return interactionId && INTERACTION_SPECS[interactionId].is_linear;
           };
 
-          $scope.isCorrectnessFeedbackEnabled = function() {
```

Comment:
Was this not used anywhere?

Reply:
No, it was. I replaced this by the passed in function with the same name.